packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -159,36 +163,38 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
      * @param depositId Unique ID corresponding to deposit order that caller wants to relay.
      * @param depositTimestamp Timestamp of Deposit emitted by L2 contract when order was initiated.
      * @param recipient Address on this network who should receive the relayed deposit.
-     * @param amount Deposited amount.
-     * @param realizedFeePct Computed offchain by caller, considering the amount of available liquidity for the token
-     * currency needed to pay the recipient and the count of pending withdrawals at the `depositTimestamp`. This fee
-     * will be subtracted from the `amount`. If this value is computed incorrectly, then the relay can be disputed.
-     * @param maxFeePct Maximum fee that L2 Depositor can pay. `realizedFee` <= `maxFee`.
-     * @param proposerRewardPct Reward % of deposit amount to pay relayers.
+     * @param amount Amount deposited on L2 to be brought over to L1.
+     * @param slowRelayFeePct Max fraction of `amount` that the depositor is willing to pay as a slow relay fee.
+     * @param instantRelayFeePct Fraction of `amount` that the depositor is willing to pay as a instant relay fee.
+     * @param quoteDeadline Timestamp up until the depositor is willing to accept an LP quotation for.
+     * @param realizedLpFeePct LP fee calculated off-chain considering the L1 pool liquidity at deposit time, before
+     *      quoteDeadline. The OO acts to verify the correctness of this realized fee. Can not exceed 50%.
      */
     function relayDeposit(
         uint64 depositId,
         uint64 depositTimestamp,
         address recipient,
         address l2Sender,
         uint256 amount,
-        // TODO: Allow caller to distinguish between slow and fast fees/rewards.
-        uint64 realizedFeePct,
-        uint64 maxFeePct,
-        uint64 proposerRewardPct
```

Comment:
note: no longer define the `proposerRewardPct`. this is baked into the `slowRelayFeePct`